sphinx/writers/latex/content.tex_t

```diff
@@ -0,0 +1,42 @@
+%% Generated by Sphinx.
```

Comment:
location under `sphinx/templates/` or `sphinx/writers/latex/templates/` would be better. (relates to #2749.)